### ISSUE TITLE
StacksAddress vs ContractAddress

### ISSUE THREAD

**Issue by friedger** [blockstack/stacks-blockchain-api#258]
**Is your feature request related to a problem? Please describe.**
I am using Connect and the client library in the same web app. There are different names used to describe the address part of a contract principal in method signatures and object properties:
* objects like `CallReadOnlyFunctionRequest` has property `stacksAddress` 
* objects like `PostConditionFungible` has property `asset.contract_address`
* parameters in Connect are named `contractAddress`, e.g. in `doContractCall`

To add to the confusion, `SmartContractApi.getContractById` uses `contractId` (address + name)

**Describe the solution you'd like**
Use `contractAddress` only when a contract should be specified by its two parts address and name.

**Describe alternatives you've considered**
Extend the documentation with a table of all different namings.

**Comment by agraebe** [comment]
This is a great one! We should aim to unify the language across tools.

Your suggestion is to use `stacksAddress` if it's just the address (e.g. `3DFJ3D`) and `contractAddress` if it is address + contract name (`3DFJ3D.contract-name`), right?

So the changes in the client lib would be:
- `contractId` > `contractAddress` ?

**Comment by friedger** [comment]
@agraebe No, do not use `stacksAddress` at all. and keep `contractId` as is. 

I have updated the issue to be more clear.

**Comment by agraebe** [comment]
Perfect, that makes sense! +1 on that!
